partcad/src/partcad/project_factory_git.py

```diff
@@ -86,6 +86,7 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
         guard_path = os.path.join(cache_path, ".partcad.git.cloned")
 
         with cache_lock:
+            git_config_params = " ".join([f"'{key}={value}'" for key, value in user_config.git_config.items()])
```

Comment:
Please extract it in separate method and cover with test. I feel like single quotes in key value could cause problems in future.